commit eb45fb2fb16df0050b9ae28f06095d72278959d9
Author: Krzysztof Kozlowski <krzk@kernel.org>
Date:   Thu Aug 27 12:53:18 2020 +0200

    docs: process: Add cross-link to security-bugs
    
    The submitting patches mentions criteria for a fix to be called
    "security fix".  Add a link to document explaining the entire process
    of handling security bugs.
    
    Signed-off-by: Krzysztof Kozlowski <krzk@kernel.org>
    Reviewed-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
    Reviewed-by: Felipe Balbi <balbi@kernel.org>
    Cc: Linus Torvalds <torvalds@linux-foundation.org>
    Cc: Kees Cook <keescook@chromium.org>
    Link: https://lore.kernel.org/r/20200827105319.9734-1-krzk@kernel.org
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>

diff --git a/Documentation/process/submitting-patches.rst b/Documentation/process/submitting-patches.rst
index 5219bf3cddfc..d5b3c5a74d5d 100644
--- a/Documentation/process/submitting-patches.rst
+++ b/Documentation/process/submitting-patches.rst
@@ -299,7 +299,8 @@ sending him e-mail.
 If you have a patch that fixes an exploitable security bug, send that patch
 to security@kernel.org.  For severe bugs, a short embargo may be considered
 to allow distributors to get the patch out to users; in such cases,
-obviously, the patch should not be sent to any public lists.
+obviously, the patch should not be sent to any public lists. See also
+:ref:`Documentation/admin-guide/security-bugs.rst <security-bugs>`.
 
 Patches that fix a severe bug in a released kernel should be directed
 toward the stable maintainers by putting a line like this::